frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,91 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'md' | 'lg'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
```

Comment:
is there a reason why we are not just overriding Chakra's base `Menu` theme and instead using our own? 
Overriding only allows users to use Chakra's base Menu item without having to import our own (if they wanted)

That also gives us a baseline of things to change.

For example, in your implementation, the focus state seems a little different from the design. Do you want a quick pair program for this?

Reply:
Thanks! edited, see if the revised implementation works